jobs/cloud-provider/templates/bin/cloud-provider_utils.erb

```diff
@@ -1,14 +1,5 @@
 #!/bin/bash -ex
 
-set_node_name() {
-  node_name="<%= spec.ip %>"
-<% if_p("cloud-provider.type") do | provider_type | %>
-  <% if provider_type == "gce" %>
-  node_name="$(curl http://metadata.google.internal/computeMetadata/v1/instance/name -H "Metadata-Flavor: Google").c.<%= p("cloud-provider.gce.project-id") %>.internal"
-  <% end %>
-<% end %>
-}
```

Comment:
We discovered this last night in the CI. Still have some issues with vSphere we will be reverting for GCE